test/unit/transformers/protocol-token-wrapper-transformer.spec.ts

```diff
@@ -21,6 +21,10 @@ describe('ProtocolTokenWrapperTransformer', () => {
   let snapshotId: string;
 
   before('Setup accounts and contracts', async () => {
+    await network.provider.request({
+      method: 'hardhat_reset',
+      params: [],
+    });
```

Comment:
It seems that coverage tool does not reset network from test-to-test, so there were some balances not being correctly accounted for. We are now resetting the state of the network on some tests.